packages/eslint-config-react-app/index.js

```diff
@@ -215,16 +201,13 @@ module.exports = {
       },
     ],
     'no-with': 'warn',
-    'no-whitespace-before-property': 'warn',
     radix: 'warn',
     'require-yield': 'warn',
-    'rest-spread-spacing': ['warn', 'never'],
     strict: ['warn', 'never'],
-    'unicode-bom': ['warn', 'never'],
     'use-isnan': 'warn',
     'valid-typeof': 'warn',
     'no-restricted-properties': [
-      'error',
+      'warn',
```

Comment:
Should stay an error. It's not about style but about the subset of features we promise to support.